apps/health/urls.py

```diff
@@ -7,12 +7,13 @@
     CheckSLSX,
 )
 
+
 urlpatterns = [
-    re_path(r"external", CheckExternal.as_view()),
-    re_path(r"external_v2", CheckExternal.as_view()),
-    re_path(r"bfd", CheckBFD.as_view()),
-    re_path(r"bfd_v2", CheckBFD.as_view()),
-    re_path(r"sls", CheckSLSX.as_view()),
-    re_path(r"db", CheckDB.as_view()),
-    re_path(r"", CheckInternal.as_view()),
+    re_path(r"external/?$", CheckExternal.as_view()),
+    re_path(r"external_v2/?$", CheckExternal.as_view()),
+    re_path(r"sls/?$", CheckSLSX.as_view()),
+    re_path(r"db/?$", CheckDB.as_view()),
```

Comment:
This matches

```
db
db/
```

but not

```
db/a
db/b
```

and so on.

But, does it match

```
db/db/
db/db/db/
```

?

I don't know, off the top of my head, if `re_path` has an implicit `^` as part of its function? I think it might not?